Javadoc formatting improvements for htsjdk.variant.* packages

james-d opened the pull request:
These changes are to documentation only, and the vast majority are just formatting changes. These are not complete (there are still some `<code>` tags that could be included) but leave the docs for the whole package in a usable state. The Javadocs generate under my Eclipse version with no errors (though there are still warnings).

james-d commented:
This is in response to issue #253

geoffjentry commented:
@james-d This is great, thanks! Could you squash the 3 commits into a single one, and then we can merge this in.

james-d commented:
Hi Jeff,

Sorry about this: I'm a bit new with github. (The reason for the multiple commits is that I was testing out the git plugin from Eclipse when I started working on this.)

I have absolutely no clue how to squash these together... In trying to do so I appear to have created a 4th commit locally:


$ git log

commit 3bebbbf75337fdf994e1362333cef886571feb92

Author: James <denvir@marshall.edu>

Date:   Sat May 16 23:16:05 2015 -0400


    Fixed some formatting issues in Javadoc comments



    Fixed some formatting issues in Javadoc comments



    Fixed Javadoc formatting in htsjdk.variant package and subpackages.



    Fixed Javadoc formatting in htsjdk.variant package and subpackages.



commit cf671d73e16a71990e61d7e6ca50910a54aa33b9

Merge: 7a8d2d7 5ec6bc5

Author: Jay Carey <jcarey@broadinstitute.org>

Date:   Wed May 13 12:20:43 2015 -0400


    Merge pull request #243 from samtools/jc_fix_travis



    Travis has been using the stale maven pom to build htsjdk.

 ...

I think one of those might be a "squashed" commit, but I can't push it back... I get an error message saying


To https://github.com/james-d/htsjdk

 ! [rejected]        master -> master (non-fast-forward)

error: failed to push some refs to 'https://github.com/james-d/htsjdk'

hint: Updates were rejected because the tip of your current branch is behind

hint: its remote counterpart. Integrate the remote changes (e.g.

hint: 'git pull ...') before pushing again.

Any chance you could just walk me through that? Should I pull from my remote repository (github.com/james-d) to my local machine, and then do something? That seems counter-intuitive.

Sorry for being high-maintenance...

Jim

From: Jeff Gentry <notifications@github.com<mailto:notifications@github.com>>
Reply-To: samtools/htsjdk <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, May 18, 2015 at 17:08
To: samtools/htsjdk <htsjdk@noreply.github.com<mailto:htsjdk@noreply.github.com>>
Cc: James Denvir <denvir@marshall.edu<mailto:denvir@marshall.edu>>
Subject: Re: [htsjdk] Javadoc formatting improvements for htsjdk.variant.* packages (#254)


@james-d<https://github.com/james-d> This is great, thanks! Could you squash the 3 commits into a single one, and then we can merge this in.

-
Reply to this email directly or view it on GitHub<https://github.com/samtools/htsjdk/pull/254#issuecomment-103211558>.

james-d commented:
I see this failed `htsjdk.samtools.util.RelativeIso8601DateTest.testLazyInstance`, which seems strange as 1. All the changes are in comments, and 2. I only made changes to the `htsjdk.variant.*` packages and two classes in the `htsjdk.tribble` package. It passed all tests before I squashed three commits together. Any ideas?

geoffjentry commented:
Huh, that's weird. The squash looks ok, and I'm suspicious of the failure as 3/4 htsjdk builds worked fine. I retriggered the failing one, let's see what happens. If it fails again, I'll take a closer look.

geoffjentry commented:
@james-d Looks like it was a transient problem. This looks great, thank you for doing this!
